cocos/core/gfx/pipeline-state.ts

```diff
@@ -151,7 +151,7 @@ export class GFXInputState {
 export interface IGFXPipelineStateInfo {
     primitive: GFXPrimitiveMode;
     shader: GFXShader;
-    inputState: GFXInputState;
+    inputAssember: GFXInputAssembler,
```

Comment:
This is unnecessary since `inputAssembler.attributes` is a valid GFXInputState

Reply:
Ok.

Reply:
Now PipelineStateManager uses `GFXRenderPass`, `GFXInputAssembler ` and `IPSOCreateInfo` to create GFXPipelineState. None of them includes `GFXInputState`.

Reply:
Ok, it is fixed.